visualization/app/codeCharta/state/metric.service.ts

```diff
@@ -63,14 +62,8 @@ export class MetricService implements FilesSelectionSubscriber, BlacklistSubscri
 	}
 
 	public getAttributeTypeByMetric(metricName: string, state: State): AttributeTypeValue {
-		const attributeType = this.getMergedAttributeTypes(state.fileSettings.attributeTypes).find(x => {
-			return _.findKey(x) === metricName
-		})
-
-		if (attributeType) {
-			return attributeType[metricName]
-		}
-		return null
+		const attributeType = state.fileSettings.attributeTypes.nodes[metricName]
```

Comment:
I think we should for now check both the nodes and edges attributeTypes as before.

Reply:
I would disagree and would instead suggest to add a similar function to the edgeMetric.service instead. That way the metric service does only node metric stuff and leaves all edge things to the dedicated edgeMetric service

Reply:
oh that's a very good idea. Didn't think about it close enough 👍

Reply:
but am I missing something or is this functionality not implemented in the edgeMetricDataService?

Reply:
You are right, this functionality is not implemented in this PR (because its functionality is not used currently). However the AttributeTypes II PR includes this method, since there attributeTypes for EdgeMetrics are used.